qutebrowser/misc/utilcmds.py

```diff
@@ -217,3 +217,20 @@ def debug_set_fake_clipboard(s=None):
         utils.log_clipboard = True
     else:
         utils.fake_clipboard = s
+
+
+@cmdutils.register()
```

Comment:
I think it'd make sense to add `hide=True` here, as this is a command which is not typically used with the commandline completion.